### ISSUE TITLE
Few issues not fixed for multiplayer

### ISSUE THREAD

**Issue by NightMare-** [ValveSoftware/halflife#1258]
PC Specs:
Processor: AMD Phenom II x6 1100T Black Edition
Motherboard: Gigabyte 990-FXA-UD3
Graphics Card: Gigabyte Radeon HD 7770 OC
Ram: 8gb DDR3

I play HLDM/AG and here are the issues:

1.) fps_override 1 - Allows players to set fps above 100. This command is great for clients to achieve smoother image quality, however, it speeds up the game resulting in "speedhack". (ex. values of 125, 250, etc...)

2.) auto bunnyhop - Since the removal of special alias, some players and admins found a way to get around this by replacing ".dll's". Even though there is server command to allow this or not, this is not enough. If the game would simply fail to launch with such modified .dll's (authentication required) then this would help solve this problem. Each and every steam user should use the original .dll. 

In conclusion, these two issues are very important. Why should a player be able to run, jump, spawn, and shoot faster with some command? Why should admins decide to allow modified .dll's? Skill wants a voice too.

Below I have attached some links. I'm showing this so you know exactly where I am coming from. I train very hard the right way and it's very discouraging to know players can do such things. I'm playing without scripts (not even bunnyhop script), default sprites/models/sound, and using 100 fps in 1920 x 1080 resolution. Nobody else plays this way, unfortunately. This game doesn't need scripts.

The full demo:
https://www.youtube.com/watch?v=l9bcReMs_NY

Gameplay montage:
https://www.youtube.com/watch?v=hHOTLJC0DWU
